src/components/data-entry/DataSetStateButton.tsx

```diff
@@ -20,9 +20,14 @@ interface DataSetStateButtonProps {
     dataSetInfo: Maybe<DataSetOpenInfo>;
 }
 
+const applyToAllMonthsMessage = i18n.t(
```

Comment:
i18n.t cannot be used at root level, because when i18n language can be changed after the file is imported. Typical workaround: move to a function.